sqlglot/generator.py

```diff
@@ -61,6 +61,9 @@ class Generator:
         exp.TsOrDsAdd: lambda self, e: self.func(
             "TS_OR_DS_ADD", e.this, e.expression, exp.Literal.string(e.text("unit"))
         ),
+        exp.TsOrDsDiff: lambda self, e: self.func(
+            "TS_OR_DS_DIFF", e.this, e.expression, exp.Literal.string(e.text("unit"))
```

Comment:
why does this and dsadd need an override?

Reply:
We probably don't need them. The `TsOrDsAdd` one was added a long time ago (😄): https://github.com/tobymao/sqlglot/commit/c8063bfdb#diff-362498362ae539e216cd83ebfe510977107648742f1a1bee7b2db7b5b7d27974

Reply:
PR no. 39

Reply:
Got rid of 'em